### ISSUE TITLE
Mosaico image upload and gallery error

### ISSUE THREAD

**Issue by hcallens** [veda-consulting/uk.co.vedaconsulting.mosaico#365]
When trying to upload an image to the Mosaico editor I get an internal server error and the image doesn't appear in the template nor in the gallery.

However, when I check the upload folder the image has actually been uploaded. Also, when I compose a traditional mail I can see the images in the KC Finder. Imagick is present on my server.

Any idea how to fix this?

I am on Wordpress 5.3.2, CiviCRM 5.17.3 and Mosaico 2.3.

**Comment by mwestergaard** [comment]
Just a guess here... 

I had a similar issue some months ago. As far as I could tell, Imagick was installed correctly and seemed fine within Wordpress by itself. I investigated with an admin at my hosting site (CiviHosting) and found that Imagick was failing with Mosaico due to insufficient CPU. The admin temporarily increased my process CPU limit and it worked. Mosaico now supports GD as a graphics driver (Administer -> CiviMail -> Mosaico Settings) and that has worked for me at the lower CPU restriction.

To be consistent, I also switched Wordpress to GD. I wrote my own extremely simple "must-use" plugin, but there are others.
